test/integration/options.spec.js

```diff
@@ -1,16 +0,0 @@
-'use strict';
-
-/**
- * Tests for cmdline options.
- * Provides umbrella for all spec files in the "options" subdirectory.
- */
-
-var path = require('path');
-var utils = require('../../lib/utils');
-
-describe('options', function() {
```

Comment:
This was better with the `describe` grouping the cmdline options...

Reply:
For consistency, "show-plugins.spec.js" should be broken into two files whose contents should be in the "options" directory...

Reply:
OK, I've split this up.  does it belong in this PR?  likely not, but sometimes it's just easier to do it or it won't get done...

Reply:
That addressed the "show-plugins.spec.js" break-up. Still don't see why this file had to go...

Reply:
ohhhh OK, I think I zapped the file because I thought it was vestigial and part of a bad merge.

I don't know why, but something about it caused the code coverage numbers to drop.  I made changes `test.node.integration` in `package-scripts.js` elsewhere to address the issue.  I think the problem may be that `options.spec.js` doesn't accurately replicate what `Mocha#loadFiles` does?  Even if we did try to do it programmatically, there's still the challenge of not being able to reach the current `Mocha` instance from the test files themselves. 
 
Even though we lose the wrapper suite, the options tests are still run in alphabetical order w/o this file. 

you're welcome to take another stab at loading things this way, as it'd be nice to have an example we can point users to.